guides/plugins/plugins/content/cms/add-cms-element-via-admin-sdk.md

```diff
@@ -0,0 +1,352 @@
+# Add CMS element via AdminExtensionSDK
+
+## Overview
+
+This article will teach you how to create a new CMS element via the AdminExtensionSDK. The plugin in this example will be named `SwagBasicAppCmsElementExample`, similar to the other guides.
+
+## Prerequisites
+
+You won't learn how to create a plugin or an app in this guide, head over to our Plugin or App Base Guide to create your first extension.
+
+This guide will also not explain how a custom component can be created in general, so head over to the official guide about creating a custom component to learn this first. It also won't go too much into detail about the AdminExtensionSDK. For that, please have a look at the [AdminExtensionSDK documentation](https://shopware.github.io/admin-extension-sdk/docs/guide/getting-started/installation).
+
+Note: This example uses TypeScript, which is recommended, but not required for developing Shopware.
+
+{% page-ref page="../../administration/app-base-guide.md" %}
+{% page-ref page="../../administration/add-custom-component.md" %}
+
+## Creating your custom element
+
+Like in the Plugin Guide for this topic, imagine you want to create a new element to display a Dailymotion video. The shop manager can configure the link of the video to be shown. That's exactly what you're going to build in this guide.
+
+Creating a new element via app requires you to use the AdminExtensionSDK.
+
+### Target structure
+
+It is yours to decide which approach to use when creating apps, since everything we're doing here will be loaded via iFrame, but since Shopware's best practice is a full VueJs approach, this tutorial will do exactly that.
```

Comment:
```suggestion
It is yours to decide which approach to use when creating apps, since everything we're doing here will be loaded via iFrame, but since Shopware's best practice is a full Vue.js approach, this tutorial will do exactly that.
```